Update КомандаУправлениеСеансами.os

6en9er opened the pull request:
Добавил 2 команды "--cluster" и "--cluster-name". Позволяет задать идентификатор кластера или его имя для ситуации, когда на сервере несколько кластеров.

EvilBeaver committed:
Merge branch 'os-dev' into oslib-master

6en9er committed:
Update КомандаУправлениеСеансами.os

Добавил 2 команды "--cluster" и "--cluster-name". Позволяет задать идентификатор кластера или его имя для ситуации, когда на сервере несколько кластеров.

EvilBeaver commented on src/Классы/КомандаУправлениеСеансами.os:
```diff
@@ -368,8 +378,39 @@
+		МассивСтрок = СтрРазделить(СписокКластеров, Символы.ПС);
+		Для Каждого Стр Из МассивСтрок Цикл
+			Если СтрНачинаетсяС(Стр, "cluster") Тогда
+				СтруктураКластера.Вставить("УИДКластера",СокрЛП(Сред(Стр, (СтрНайти(Стр, ": ") + 2),СтрДлина(Стр))));
```
Давай вынесем эти вычисления строк Сред, СтрНайти, плюс 2 и прочее в отдельный метод

6en9er replied:
Вынес в функцию

6en9er committed:
Update КомандаУправлениеСеансами.os

Вынес расчет строки значения имени и идентификатора кластера в отдельную функцию
